pubsub/src/main/java/com/github/twitch4j/pubsub/domain/PredictionResult.java

```diff
@@ -0,0 +1,26 @@
+package com.github.twitch4j.pubsub.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+@Data
+@Setter(AccessLevel.NONE)
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PredictionResult {
+
+    private String type; // e.g. WIN or LOSE
```

Comment:
Can we use a enum for this too (WIN/LOSE)?

Reply:
I believe there could be at least one more state for this variable as well; if a prediction isn't resolved within 24 hours, the points are refunded, so `type` may be something special in this case

Reply:
Alright, on 2nd thought maybe using strings for states on unofficial topics is a better choice in general in case they change states unannaunced - so this would be more resilient to changes.